Feature - import API Management policy to a product script (#58)
arcus-azure/arcus.scripting: docs/preview/features/powershell/azure-api-management.md

| `Method`            | yes       | The method of the to-be-created operation on the API                                                     |
| `UrlTemplate`       | yes       | The URL-template, or endpoint-URL, of the to-be-created operation on the API                             |
| `OperationName`     | no        | The optional descriptive name to give to the to-be-created operation on the API (default: `OperationId`) |
| `Description`       | no        | The optional explanation to describe the to-be-created operation on the API                              |
| `PolicyFilePath`    | no        | The path to the file containing the optional policy of the to-be-created operation on the API            |

**Example**

Creates a new API operation on the Azure API Management service with using the default base operation policy.

```powershell
PS> Create-AzApiManagementApiOperation -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ApiId $ApiId -OperationId $OperationId -Method $Method -UrlTemplate $UrlTemplate
# New API operation '$OperationName' on Azure API Management service was added.
```

> Creates a new API operation on the API Management service with using a specific operation policy.

Reviewer: ```suggestion
Creates a new API operation on the Azure API Management instance with using a specific operation policy.
```

PS> Create-AzApiManagementApiOperation -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ApiId $ApiId -OperationId $OperationId -Method $Method -UrlTemplate $UrlTemplate -OperationName $OperationName -Description $Description -PolicyFilePath $PolicyFilePath
# New API operation '$OperationName' on API Management service was added.
```	

> ## Import policy to an product in the API Management service

Reviewer 2: ```suggestion
## Import policy to a product in the API Management service
```

> Imports a policy from a file to an product in Azure API Management.

Reviewer 2: ```suggestion
Imports a policy from a file to a product in Azure API Management.
```

| Parameter           | Mandatory | Description                                                                                              |
| ------------------- | --------- | -------------------------------------------------------------------------------------------------------- |
| `ResourceGroupName` | yes       | The resource group containing the API Management service                                                 |
| `ServiceName`       | yes       | The name of the API Management service located in Azure                                                  |

> | `ProductId`         | yes       | The ID to identify the product running in API Management                                                     |

Reviewer 2: ```suggestion
| `ProductId`         | yes       | The ID to identify the product in API Management                                                     |
```

> # New API operation '$OperationName' on API Management service was added.

Reviewer: This looks like a doc bug or not?
```suggestion
```

Author: No, it's the powershell comment that is here presented as the output of the command 😃 .

> ```	

Reviewer: This looks like a doc bug or not?
```suggestion
```

Author: No, it's the powershell comment that is here presented as the output of the command 😃 .

| ------------------- | --------- | --------------------------------------------------------- |
| `ResourceGroupName` | yes       | The resource group containing the Azure API Management service  |
| `ServiceName`       | yes       | The name of the Azure API Management service located in Azure   |
| `ProductId`         | yes       | The ID to identify the product in Azure API Management          |
| `PolicyFilePath`    | yes       | The path to the file containing the to-be-imported policy |

```powershell
PS> Import-AzApiManagementProductPolicy -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ProductId $ProductId -PolicyFilePath $PolicyFilePath
# Updating policy of the product '$ProductId'
```

## Import policy to an API in the Azure API Management service

Imports a base-policy from a file to an API in Azure API Management.

| Parameter           | Mandatory | Description                                               |
| ------------------- | --------- | --------------------------------------------------------- |
| `ResourceGroupName` | yes       | The resource group containing the Azure API Management service  |

> | `ServiceName`       | yes       | The name of the API Management service located in Azure   |

Reviewer: ```suggestion
| `ServiceName`       | yes       | The name of the Azure API Management instance located in Azure   |
```
